### ISSUE TITLE
Tiling effect and "black hole" artifacts

### ISSUE THREAD

**Issue by z-pan** [cabooster/DeepCAD#12]
Hi cabooster, 

I tested with the pretrained model G_12_1199.pth on my microscopy image data, and the output images show a patch edge effect (please see the image below, left: input; right: output).
![image](https://user-images.githubusercontent.com/12038796/232184686-b69575ac-e3a7-4a6f-a52d-8942536fc8e5.png)
Also, I could observe "black hole"-like artifacts, i.e., some strange black circle-shaped artifacts (please see the image below, left: input; right: output).
![image](https://user-images.githubusercontent.com/12038796/232184852-b15ce48b-8cb6-4325-9836-1dede51a4a93.png)

Thank you!

**Comment by z-pan** [comment]
The related testing parameters for the above outputs are: --img_h 64 --img_w 64 --img_s 35 --gap_h 45 --gap_w 45 --gap_s 20.
I tried multiple combinations of the above parameters, and the issues persist. 
I also trained with my own data (similar to the shown input images), with the following training params: --lr 0.00005 --img_h 64 --img_w 64 --img_s 50 --gap_h 40 --gap_w 40 --gap_s 30 --n_epochs 25 --GPU 0 --normalize_factor 1 --train_datasets_size 4200 --select_img_num 300. However, the artifacts still persist.
